src/styles.css

```diff
@@ -183,12 +194,56 @@ button {
   opacity: 0;
 }
 
+/* --------------- MOBILE CONTROLS --------------- */
+
+.mobile-controls {
+  position: fixed;
+  bottom: 30px;
+  left: 0;
+  width: 100vw;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.mobile-controls button {
+  background: none;
+  color: rgba(255, 255, 255, 0.3);
+  width: 5rem;
+  height: auto;
+  transition: transform 0.2s ease-in-out;
+}
+.mobile-controls button:active {
+  transform: scale(0.9);
+}
+.mobile-controls button svg {
+  pointer-events: none;
+  touch-action: none;
+}
+.mobile-controls .left-turn,
+.mobile-controls .right-turn {
+  -webkit-user-select: none;
```

Comment:
These are all redundancy, `user-select` should cover all cases. It isn't like we should have to cover browsers from a decade ago(Luckily)